zig-demo-app/README-zig.adoc

```diff
@@ -0,0 +1,90 @@
+= Zig demo
+
+This is demo application showing how to create ConfD Application in Zig
+programming language.
+
+https://ziglang.org
+
+Demo implements simple action and demonstrates, how to:
+
+initialize application
+register action
+perform ConfD poll loop
+
+It also shows how to do Zig based logging and error handling.
+It does not use Zig too much, as implementation is empty, can be seen as starting
+point for ConfD application in Zig.
+
+
+== Dependencies
+
+Install zig: https://ziglang.org/learn/getting-started/#direct-download
+
+== Integrate with C
+
+Import C symbols under namespace (e.g. `c_imp`):
+----
+const c_imp = @cImport({
+    @cInclude("confd_lib.h");
+    @cInclude("confd_dp.h");
+});
+----
+
+NOTE:: Under `c_imp` namespace there are other C imports included by imported header files,
+e.g. `c_imp.stderr`. They are not identical if imported in other `@cImport` statement (other namespace), ...
+
+Usage of imported symbols:
+----
+var debuglevel: c_uint = c_imp.CONFD_TRACE;
+c_imp.confd_init(EXAMPLE_APP_NAME, c_imp.stderr, debuglevel);
+----
+
+=== C code translation
+
+C code can be translated as starting sample (`goto` and labels should not be used in C code!).
+Translated code is good start, but it has many unnecessary constructs (pointer casts, C lib usage, no error handling) and imports that can be cleaned.
+
```

Comment:
What are you trying to say here? That It's possible to auto translated to zig? If so I think you should say "auto translate". Also, I assume you're saying that the translator can't handle gotos and and labels, I think that could be expressed clearer too.

Reply:
README was updated.